gems/aws-sdk-core/lib/aws-sdk-core/shared_config.rb

```diff
@@ -6,36 +6,33 @@ class SharedConfig
     class << self
       # Constructs a new SharedConfig. SharedConfig is a hash of profile
       # to hash of configuration values. This will load the shared
-      # credentials file, and optionally the shared configuration file, as ini
+      # configuration file, and optionally the shared credentials file, as ini
       # files which support profiles.
       #
-      # By default, the shared credentials file (the default path for which is
-      # `~/.aws/credentials`) and the shared config file (the default path for
-      # which is `~/.aws/config`) are loaded. You can specify the shared
-      # credentials file path with the `ENV['AWS_SHARED_CREDENTIALS_FILE']`
-      # environment variable or with the `:credentials_path` option. Similarly,
-      # you can specify the shared config file path with the
-      # `ENV['AWS_CONFIG_FILE']`environment variable or with the `:config_path`
-      # option.
+      # By default, the shared configuration file (the default path for which is
+      # `~/.aws/config`) and the shared credentials file (the default path for
+      # which is `~/.aws/credentials`) are loaded. You can specify the shared
+      # configuration file path with the `ENV['AWS_CONFIG_FILE']` environment
+      # variable or with the `:config_path` option. Similarly, you can specify
+      # the shared credentials file path with the
+      # `ENV['AWS_SHARED_CREDENTIALS_FILE']`environment variable or with the
+      # `:credentials_path` option.
```

Comment:
Might as well update the documentation syntax in these files (from backticks to using `+` given our recent discussion about it)

Reply:
Possibly. I think because of AWS documentation using html we may need markdown support. We should check how other stuff renders.